dataclasses_avroschema/fields/pydantic_fields.py

```diff
@@ -209,3 +209,12 @@ def avro_type(self) -> typing.Dict:
 
     def fake(self) -> uuid.UUID:
         return uuid.uuid5(namespace=uuid.NAMESPACE_URL, name=fake.pystr())
+
+
+class ConstrainedIntField(PydanticField):
+    @property
+    def avro_type(self) -> typing.Dict:
+        return {"type": INT, "pydantic-class": "ConstrainedInt"}
+
+    def fake(self) -> int:
+        return 1
```

Comment:
This can be done doing something like: `return fake.pyint(min_value=self.type.gt, max_value=...). You can explore the `self.type` which is a `ConstrainedIntValue` which has thinks like `{'strict': False, 'gt': 100, 'ge': None, 'lt': None, 'le': None, 'multiple_of': None, ...}` and then you will have to apply some logic